### PR TITLE
Update pt-BR.json

### PR DESCRIPTION
**Pull request by idAlencar**

### CODE REVIEW

File: localization/languages/pt-BR.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         /* Context menu items
         these are the items displayed in the menu when you right-click on a page
         */
-        "pictureInPicture": null, //missing translation
+        "pictureInPicture": "Picture in picture", //missing translation
```
Comment (PalmerAL) [inline localization/languages/pt-BR.json]: I had the same question on #927 actually: is there not a Portuguese translation for this?

**Comment by PalmerAL** [comment]
Could you please also translate the "enableBlocking" and "disableBlocking" strings?

**Comment by idAlencar** [comment]
we use it that way (Picture in picture), the translation gives us another idea -> picture can mean (image, figure, photo), fica algo como (image in image).

"enableBlocking" and "disableBlocking" -> went unnoticed, but I will kkkkkkk

**Comment by PalmerAL** [comment]
Thanks for the explanation!

**Pull request merged**